instrumented/nightly-tests/src/main/kotlin/com/datadog/android/nightly/activities/WebViewTrackingActivity.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.nightly.activities
+
+import android.os.Bundle
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import androidx.appcompat.app.AppCompatActivity
+import com.datadog.android.nightly.BuildConfig
+import com.datadog.android.nightly.R
+import com.datadog.android.nightly.utils.measure
+import com.datadog.android.webview.DatadogEventBridge
+
+internal open class WebViewTrackingActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.webview_tracking_activity)
+        val webView: WebView = findViewById(R.id.webview)
+        webView.webViewClient = WebViewClient()
+        webView.settings.javaScriptEnabled = true
+        setupWebView(webView)
+        webView.loadUrl(
+            "https://datadoghq.dev/browser-sdk-test-playground/" +
+                "?client_token=${BuildConfig.NIGHTLY_TESTS_TOKEN}" +
+                "&application_id=${BuildConfig.NIGHTLY_TESTS_RUM_APP_ID}" +
```

Comment:
nit: To my understanding, this can be any `client_token` and any `application_id`. The native SDK should capture all events and send them using its own credentials. I think it could be healthier for E2E tests to randomize values for this query - this way we ensure that Browser SDK is unable to bypass native bridge and send any data to E2E org.

Reply:
Yeah, I think it's better to avoid any problems.